src/Lecture1.hs

```diff
@@ -119,4 +138,19 @@ and lower than 6 elements (4, 5, 6, 7, 8 and 9).
 
 🕯 HINT: Use recursion to implement this function.
 -}
-lowerAndGreater n list = error "TODO"
+
+lowerAndGreater :: Int -> [Int] -> [Char]
+lowerAndGreater number arr = printOut number greaterNum lowerNum
+    where lowerAndGreaterArr :: Int -> [Int] -> [Int]
+          lowerAndGreaterArr _ []     = []
+          lowerAndGreaterArr n [x]    | x > n = [1] | x < n = [-1] | otherwise = [0]
+          lowerAndGreaterArr n (x:xs) = lowerAndGreaterArr n [x]  ++ lowerAndGreaterArr n xs
+
+          numTimesFound :: Eq a => a -> [a] -> Int
+          numTimesFound x = length . filter (== x)
```

Comment:
That's correct! 💯 

However, with this solution, you will walk through the list twice.
With the recursive solution and manual check on every element in the list on both conditions at the same time (without `filter` function), you would be able to write it more performant with only one traversal of the list 🙂